tests/rustworkx_tests/test_random.py

```diff
@@ -143,10 +143,10 @@ def test_random_gnm_undirected_complete_graph(self):
 
     def test_random_gnm_undirected_invalid_num_nodes(self):
         with self.assertRaises(ValueError):
-            rustworkx.undirected_gnm_random_graph(-23, 5)
+            rustworkx.undirected_gnm_random_graph(0, 5)
 
     def test_random_gnm_undirected_invalid_probability(self):
-        with self.assertRaises(ValueError):
+        with self.assertRaises(OverflowError):
```

Comment:
PyO3 returns an overflow error for passing a negative to an unsigned type input I would have done a type error, but I guess this makes sense too. But anyway I think we need a release note documenting this as a potentially breaking change. I don't think it's a severe API change and something we should do to simplify the code, but we should just document it as a change.

Reply:
I can change it to a TypeError. What category in the release note should this go under? Upgrade? And you want to document all the input changes from isize to usize, like num_nodes and num_edges as well?

Reply:
Nah this is fine, upgrade note is the right category for this because it's something an end user might need to change when upgrading. We don't need to document the input change, it should be mostly invisible, it technically would allow them to pass in bigger bigger integers but that would fail anyway because of the max index size is a 32bit int and will error if you try to use >~4 billion nodes or edges.

Reply:
Actually I just looked at this and this test shouldn't be here. It's testing probability for the `gnm` graph which doesn't take probability as an input. I think we can just remove this test. I can still do the reno for probability on the `gnp` graph.